lib/App/pmuninstall.pm

```diff
@@ -309,6 +309,7 @@ sub is_local_lib {
 
     my $local_lib_base = quotemeta File::Spec->catfile(Cwd::realpath($self->{local_lib}));
     $file = File::Spec->catfile($file);
+    $file = Cwd::realpath($file);
```

Comment:
I'm not sure why the original code is using `catfile` with one file argument. Is it to get the call to `canonpath` called by [`catfile`](https://metacpan.org/source/XSAWYERX/PathTools-3.75/Cwd.xs#L715)?

Perhaps just call that directly as:

```perl
File::Spec->canonpath(Cwd::realpath(...))
```

Reply:
@zmughal Yes, perhaps you are correct.